pynamodb/attributes.py

```diff
@@ -386,6 +387,89 @@ def serialize(self, value):
             return six.u(value)
 
 
+class MultiKeyMap(object):
+    def __init__(self):
+        self.key = None
+        self.value = None
+        self.keymap = {}
+
+    def translate_key(self, key):
+        return self.keymap.get(key, key)
+
+    def add_keys(self, actual_key, keys):
+        self.keymap[actual_key]=actual_key
+        for key in keys:
+            self.keymap[key] = actual_key
+
+    def __getitem__(self, key):
+        if self.key != self.translate_key(key):
+            return None
+        return self.value
+
+    def __setitem__(self, key, value):
+        self.key = self.translate_key(key)
+        self.value = value
+
+    def __get__(self, instance, owner):
+        if instance:
+            return self.value
+        else:
+            return self
+
+    def get_key_value(self):
+        return (self.key, self.value)
+
+    def get_value(self):
+        return self.value
+
+class EnumAttribute(Attribute):
```

Comment:
FYI we have lots of code that create `EnumAttribute` to represent python Enum either by name or value